src/FsAutoComplete.Core/FsAutoComplete.Core.fsproj

```diff
@@ -13,6 +13,11 @@
   <PropertyGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
     <DefineConstants>NO_EXTENSIONTYPING;$(DefineConstants);DOTNET_SPAWN</DefineConstants>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="DotnetNewTemplate.fs" />
+    <Compile Include="Debug.fs" />
+    <ProjectReference Include="..\FsAutoComplete.Core.VerboseSdkHelper\FsAutoComplete.Core.VerboseSdkHelper.fsproj" />
```

Comment:
i dont expect to see `Debug.fs` and that project reference `FsAutoComplete.Core.VerboseSdkHelper` in the diff

Reply:
Sure. They have been removed now.